developer-workflow/grammar.rst

```diff
@@ -31,15 +31,16 @@ Note: sometimes things mysteriously don't work.  Before giving up, try ``make cl
   (This runs Python's parser generator, ``Tools/peg_generator``).
```

Comment:
Maybe we could do

```suggestion
  (This runs Python's parser generator, :cpy-file:`Tools/peg_generator`).
```

since `blob` redirects to `tree` if needed?